sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/SyntheticDataPublisher.java

```diff
@@ -50,15 +54,22 @@
  * <pre>
  *  ./gradlew :beam-sdks-java-load-tests:run -PloadTest.args='
  *    --insertionPipelineTopic=TOPIC_NAME
+ *    --kafkaBootstrapServerAddress=SERVER_ADDRESS
+ *    --kafkaTopic=KAFKA_TOPIC_NAME
  *    --sourceOptions={"numRecords":1000,...}'
- *    -PloadTest.mainClass="org.apache.beam.sdk.loadtests.SyntheticDataPubSubPublisher"
+ *    -PloadTest.mainClass="org.apache.beam.sdk.loadtests.SyntheticDataPublisher"
  *  </pre>
+ *
+ * If parameters related to Kafka are provided, the publisher writes to Kafka. If both pubsub topic
+ * and Kafka params are present, records will be written to both sinks.
```

Comment:
It would be even more clear if the sentence gives instructions for both Kafka and Pubsub, so I suggest: 

```
If parameters related to a specific sink are provided (Kafka or PubSub), the pipeline writes to the sink. Writing to both sinks is also acceptable.
``` 
(Or similar but including both runners)